### DISCUSSION TITLE
Equipo Hackathon?

### DISCUSSION THREAD

**Discussion by dev-josegarcia** [Launch-X-Latam/MisionFrontEnd#591] [Team Posts]
Faltan 2 integrantes para formar equipo hackathon quien se une?

**Comment by AnzeuS** [comment]
Buenas quisiera unirme 

Te paso mi correo andrewsloranunez@gmail.com

**Comment by Fco-Alvarez** [comment]
Yo también estoy sin equipo XD, correo de innovacccion ***@***.*** Enviado desde mi iPhoneEl 30-12-2022, a la(s) 14:39, AnzeuS ***@***.***> escribió:﻿
Buenas quisiera unirme

**Comment by dev-josegarcia** [comment]
pasame tu correo para contactar.

**Comment by edgardorojas** [comment]
Estoy en busca de equipo, mi correo lx17007@innovaccion.mx. Saludos, felices fiestas

**Comment by Drako9159** [comment]
Aún hay espacio?

**Comment by LuisSerranoGtz** [comment]
lx02079@innovaccion.mx, aun hay espacio?

**Comment by KarenPamelaLopezMedina** [comment]
¿cuándo será el hackaton?
¿Ayuda saben cuando responderá Zoe?

**Comment by shelkanife** [comment]
Ando buscando team, este es mi correo -> lx15879@innovaccion.mx

**Comment by gumodi35** [comment]
Me interesa, confirmen si hay espacio! pueden mandarme mensaje por DM en mi profile!

**Comment by joseandresgavilanes** [comment]
Busco equipo  ***@***.***

**Comment by JocelinFlores** [comment]
Busco equipo para hackaton aun hay espacio?

**Comment by shelkanife** [comment]
> Busco equipo para hackaton aun hay espacio?

Si gustas podemos formar un equipo con las personas que sobran en esta publicación, ¿qué les parece?

**Comment by gumodi35** [comment]
Si me parece, y si tienen otro medio de contacto, por qué los hilos en esta conversación son intermitentes, para no perder a los interesados ✅

**Comment by RicardoLunaRivera** [comment]
me puedo unir, dejo mi correo: lx04128@innovaccion.mx
Saludos.

**Comment by Fcovaldz** [comment]
Yo igual busco equipo, dejo mi correo lx00111@innovaccion.mx

**Comment by RicardoLunaRivera** [comment]
> Busco equipo para hackaton aun hay espacio?
> 
> Si gustas podemos formar un equipo con las personas que sobran en esta publicación, ¿qué les parece?

Yo entro, les dejo mi correo para ponernos deacuerdo: lx04128@innovaccion.mx, ya sea para unirme a un grupo o alguien quiera crear uno nuevo.

**Comment by JocelinFlores** [comment]
Yo tambien dejare por aqui mi correo y como comentan si tenemos otro medio de contacto no se perderan entre tantos hilos lx02315@innovaccion.mx

**Comment by AlexGeovanni** [comment]
dejare mi correo por si hay un lugar lx00396@innovaccion.mx

**Comment by SNOOPYFC** [comment]
lx00617@innovaccion.mx
agregame porfa

**Comment by joseandresgavilanes** [comment]
busco equipo

**Comment by Atzin99m** [comment]
Mi correo es lx03322@innovaccion.mx o atzin.luna.ramirez@upvm.edu.mx para que me puedan por favor agregar a su equipo.
